installer/win/Symphony-x64.aip

```diff
@@ -20,7 +20,6 @@
     <ROW Property="BannerBitmap" Value="banner" MultiBuildValue="DefaultBuild:Banner.jpg" Type="1" MsiKey="BannerBitmap"/>
     <ROW Property="CTRLS" Value="2"/>
     <ROW Property="DialogBitmap" Value="dialog" MultiBuildValue="DefaultBuild:Tabloid.jpg" Type="1" MsiKey="DialogBitmap"/>
-    <ROW Property="MINIMIZE_ON_CLOSE" Value="CheckBox"/>

```

Comment:
why are we getting rid of property, i thought we just wanted to change default from true to false, on line 153.